Decide.java

```diff
@@ -125,8 +125,28 @@ private static boolean CMV2(){
         
     }
 
-    private static boolean CMV3(){
-        //TODO
+    private static boolean CMV3(double area1){
+        double x1 = x[0]; 
+        double y1 = y[0]; 
+        double x2 = x[1]; 
+        double y2 = y[1]; 
+        for(int i= 2; i < NUMPOINTS; ++i){

```

Comment:
Very nitpicky, but sometimes try to keep a consistent codebase. You have line breaks in other functions above loops while here you don't